files/preseeds/Debian.preseed

```diff
@@ -54,11 +54,36 @@ d-i clock-setup/ntp boolean true
 
 # Partitioning
 d-i partman-auto/disk string /dev/sda
-#d-i partman-auto/method string regular
+d-i partman-basicfilesystems/no_swap boolean false
+d-i partman-auto/choose_recipe select myroot
+d-i partman-auto/expert_recipe string \
```

Comment:
This looks painful! :)

Does that work on all machines?

Reply:
Ran and confirmed to work on all bench machines !

Reply:
Joys of preseeds bathe us in eternal happiness....